### PR TITLE
Problem: enabling eslint configuration uncovers violations

### PR DESCRIPTION
**Pull request by lenards**
This pull request fixes the violations uncovered #56. This rule violations is remarkably minor, they were nearly all `no-unused-vars` violations (one was a typo in variable name, but appeared as an "unused" variables).

### CODE REVIEW

File: src/styles/pad.js [hunk 1]
```diff
@@ -36,7 +34,7 @@ const Pad = ( props ) => {
                     : value
                 style.paddingLeft = typeof value == "number"
                     ? sizes[value -1]
-                    : vlaue
```
Comment (Tharon-C) [inline src/styles/pad.js:39]: Doh! Case for writing tests.

Comment (lenards) [reply src/styles/pad.js:39]: Yup.

File: src/FlatButtonConfirm.js [hunk 2]
```diff
@@ -45,7 +45,6 @@ export default class PopoverExampleSimple extends React.Component {
             confirmLabel,
             content,
             style,
-            disabled,
```
Comment (Tharon-C) [inline src/FlatButtonConfirm.js:48]: +1 for tests. I should have passed this prop to the control button. We can leave, I'll catch it again when creating a guide example for this. It's a wip component.

**Comment by Tharon-C** [comment]
LGTM :+1:
:shipit:

**Pull request merged**